rct229/rulesets/ashrae9012019/section1/section1rule1.py

```diff
@@ -0,0 +1,142 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import (
+    BASELINE_0,
+    BASELINE_90,
+    BASELINE_180,
+    BASELINE_270,
+    PROPOSED,
+    USER,
+)
+from rct229.rulesets.ashrae9012019.data_fns.table_4_2_1_1_fns import (
+    table_4_2_1_1_lookup,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_BPF_building_area_types_and_zones import (
+    get_BPF_building_area_types_and_zones,
+)
+from rct229.utils.assertions import assert_
+from rct229.utils.jsonpath_utils import find_one
+from rct229.utils.pint_utils import ZERO
+
+MANUAL_CHECK_REQUIRED_MSG = "One or more building area types could not be determined for the project's building segments. Assigning a lighting building area type to all building segments will fix this issue."
+
+
+class Section1Rule1(RuleDefinitionListIndexedBase):
+    """Rule 1 of ASHRAE 90.1-2019 Appendix G Section 1 (Performance Calculations)"""
+
+    def __init__(self):
+        super(Section1Rule1, self).__init__(
+            rmrs_used=produce_ruleset_model_instance(
+                USER=True,
+                BASELINE_0=True,
+                BASELINE_90=True,
+                BASELINE_180=True,
+                BASELINE_270=True,
+                PROPOSED=True,
+            ),
+            rmrs_used_optional=produce_ruleset_model_instance(
+                USER=True,
```

Comment:
We expect USER, PROPOSED, AND BASELINE_0  RMDs always exist. This means we don't need lines 39,40, and 44.

Reply:
After deleting, this rule will fail. Juan is working on this and will provide us with the updated RMDs.